contracts/EIP-6366/EIP-6366-Core.sol

```diff
@@ -1,12 +1,12 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.7;
-import './interfaces/IERC6366Error.sol';
-import './interfaces/IERC6366Core.sol';
+import './interfaces/IEIP-6366-Error.sol';
```

Comment:
I think it's weird to add a `-` in the file name and not in the contract name
We should keep the name like `IEIP6366Error.sol`

Reply:
I see the CI/CD of ethereum/EIPs was failed due to the checking for related link.
https://github.com/ethereum/EIPs/actions/runs/4092373742/jobs/7057063640